### PR TITLE
NEW: Implement full JSON output, without changing existing logic

### PR DESCRIPTION
**Pull request by mifkata**
## Overview

This PR is about changing the output format from `Table` to `JsonOutput` (a simplified Json formatter). It doesn't change anything about current logic in regards to data fetching and sorting. The only difference is that:

* All other outputs are silenced, so you can `corona --json > output.json`
* Changes the format of numbers to not use `comma-number`, but instead return plain integers
* When `cli.flags.json === true` all coloring functions in `utils/table.js` will return plain text instead
* Outputs JSON

## How to test
* `corona --json` - normal run
* `corona -j` - same as above using short-handle 
* `corona --json > output.json` - output to `output.json` and check the formatting (Prettifying JSON helps)

### CODE REVIEW

**Comment by mifkata** [comment]
@ahmadawais Hello, just a friendly reminder to review this :) Thank you!

File: index.js [hunk 1]
```diff
@@ -38,11 +39,12 @@ const chart = cli.flags.chart;
...
 const bar = cli.flags.bar;
 const minimal = cli.flags.minimal;
-const options = { sortBy, limit, reverse, minimal, chart, log, bar };
+const json = cli.flags.json;
+const options = { sortBy, limit, reverse, minimal, chart, log, json, bar };
 
 (async () => {
 	// Init.
-	await init(minimal);
+	init(minimal || json);
```
Comment (ahmadawais) [inline index.js:47]: `await` is needed so the update check will run before the spinner.

**Review by ahmadawais — APPROVED** [review]
Looks good to me! ✔︎

**Comment by ahmadawais** [comment]
🔗 https://github.com/ahmadawais/corona-cli/releases/tag/5.1.0
🔗 https://www.npmjs.com/package/corona-cli

Thanks a lot @mifkata for working on this.

One improvement I can think of is that the keys for JSON could be

`country`, `cases`, `'cases-today'`, `deaths`, `'deaths-today'`, `recovered`, `active`, `critical`, `'per-million'`.

**Comment by mifkata** [comment]
> One improvement I can think of is that the keys for JSON could be
> 
> `country`, `cases`, `'cases-today'`, `deaths`, `'deaths-today'`, `recovered`, `active`, `critical`, `'per-million'`.

Do you want me to open a PR for it @ahmadawais ? :)

**Comment by ahmadawais** [comment]
Sure thing.

>

**Pull request merged**